simple-api/kurtosis-package/main.star

```diff
@@ -5,7 +5,6 @@ KURTOSIS_API_SERVICE_NAME = "kurtosis-simple-api"
 KURTOSIS_API_IMAGE_NAME = "kurtosistech/kurtosis-simple-api"
 KURTOSIS_API_PORT_NAME = "http"
 
-
```

Comment:
Haha, I added this line previously (and I think it was actually Gyani who asked me to add it). I think it's part of Python general style guide to separate root functions with 2 blank lines

Reply:
Oh cool. I'll add it back haha.